packages/cli/src/ampx.ts

```diff
@@ -46,5 +46,8 @@ try {
   if (e instanceof Error) {
     printer.log(format.error('Failed to emit usage metrics'), LogLevel.DEBUG);
     printer.log(format.error(e), LogLevel.DEBUG);
+    if (e.stack) {
+      printer.log(format.error(e.stack), LogLevel.DEBUG);
```

Comment:
`format.error` is going to make the whole stack trace red which personally I find difficult to read. I'd prefer just
```suggestion
      printer.log(e.stack, LogLevel.DEBUG);
```